### ISSUE TITLE
Sometimes total row includes machines extended description

### ISSUE THREAD

**Issue by USSX-Hares** [factoriolab/factoriolab#1414]
**Describe the bug**
![image](https://github.com/factoriolab/factoriolab/assets/29100587/3447c9e1-ac95-4e29-b2e7-7a431c748190)

**To Reproduce**
Reproduced occasionally multiple times. Actual StR not determined yet.

**Desktop:**

 * OS: Windows 10
 * Browser: Chrome
 * Version: 126.0.6478.114

**Comment by dcbroad3** [comment]
I've noticed this as well and can't figure out what's causing it. If you come up with steps to reproduce that would be awesome, I've tried to look into it with no luck so far.

**Comment by dcbroad3** [comment]
Seems like I'm able to reproduce this pretty consistently by hovering to show some of the tooltips, then changing the preset in settings. I've tested a few tweaks locally, and it seems like if I dumb down the tracking function for those rows, the issue goes away. Something about the way it's reorganizing those rows based on the tracking function seems like it must be broken. 

Merged a change that will hopefully fix this.
